applications/ParachainPulse.md

```diff
@@ -0,0 +1,219 @@
+| 0e. | Article | We will publish an article that explains what was achieved as part of the grant and how users can benefit the most from ParachainPulse functionalities. |
+| 1. | Implement Github scraper functionality | Periodically update the list of repositories that are being part of the Polkadot & Kusama and each of their parachains on GitHub |
+| 2. | Implement Github scraper functionality | Calculate the total number of commits, repositories and contributors within the Polkadot & Kusama and each of their parachains |
+| 3. | Implement Github scraper functionality | Get the contributors of the month based on the number of commits for Polkadot & Kusama and each of their parachains |
+| 4. | Implement Github scraper functionality | Calculate the total number of commits across all the repositories for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 5. | Implement Github scraper functionality | Calculate the number of active repositories for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 6. | Implement Github scraper functionality | Calculate the number of active contributors for each month over the last year for Polkadot & Kusama and each of their parachains |
+| 7. | Implement Github scraper functionality | Collect the list of recent commits across all the Polkadot's Github repositories from the last 30 days |
+| 8. | Implement the ParachainPulse APIs required by frontend | Statistics API that returns the number of monthly active projects, monthly active developers and monthly contributions for Polkadot & Kusama |
+| 9. | Implement the ParachainPulse APIs required by frontend | Growth API that returns the active monthly growth for developers, projects and contributions over the last 12 months for Polkadot & Kusama and each of their parachains |
+| 10. | Implement the ParachainPulse APIs required by frontend | Developers API that returns the list of contributors, the number of contributors and active contributors for Polkadot & Kusama and each of their parachains |
+| 11. | Implement the ParachainPulse APIs required by frontend | Projects APIs that return the list of projects, the total number projects, the active projects during the last 30 days and during the last 12 months for Polkadot & Kusama and each of their parachains |
+| 12. | Implement the ParachainPulse APIs required by frontend | Contributions API that returns the total number of commits, the number of commits during the last 30 days, and during the last 6 months and commits growth for 12 months for Polkadot & Kusama and each of their parachains |
+| 13. | Implement the ParachainPulse APIs required by frontend | Submit parachain API |
+| 14. | Prepare the final adjustments | Add Google Analytics to track users engagement in order to adapt and improve accordingly |
+| 15. | Prepare the final adjustments | Create the production infrastructure on Google Cloud |
```

Comment:
We typically only fund PoCs and not deployment to prod, so I don't know if the cloud infrastructure is something that can be covered under the grants program. Would you be willing to reduce the scope/cost to a PoC and remove this deliverable along with the deployment?

Reply:
Sure, by removing the "Prepare the final adjustments" deliverable, the M2 cost will be reduced to $28.750